python/tank/bootstrap/cached_configuration.py

```diff
@@ -239,6 +241,10 @@ def update_configuration(self):
                     os.path.join(self._path.current_os, "install", "core")
                 )
                 log.debug("Previous core restore complete...")
+        else:
+            # remove backup folders now that the update has completed successfully
+            filesystem.delete_folder(os.path.dirname(config_backup_path))
```

Comment:
@manneohrstrom This is because config_backup_path variable has a path that has a 'config' subfolder appended at the end, for example:

config_backup_path is
c:\users\gomezll\appdata\local\temp\tankTemporaryTestData_5d4ef9b4-a288-4296-bcf
1-6a0907613240\unit_test_mock_sg\site.backup_tests\cfg\install\config.backup\201
70618_211514\config

Where as core_backup_path is:
c:\users\gomezll\appdata\local\temp\tankTemporaryTestData_5d4ef9b4-a288-4296-bcf
1-6a0907613240\unit_test_mock_sg\site.backup_tests\cfg\install\core.backup\20170
618_211514

See how the path variable you are questioning has the config child folder appended at the end.

If you look further above, you can see that variable is being computed by ConfigurationWriter.move_to_backup function.

Do you want that behavior changed?